docs/kubernetes/README.md

```diff
@@ -155,6 +155,38 @@ During the initialization of the CSI controller, the default location for all vo
 3. If neither of the above is set, the `KUBE_NODE_NAME` environment variable defaults to the name of the node where the CSI controller is scheduled. This node name is then used to query the Hetzner API for a matching server and its location.
 4. As a final fallback, the [Hetzner metadata service](https://docs.hetzner.cloud/#server-metadata) is queried to obtain the server ID, which is then used to fetch the location from the Hetzner API.
 
+### Volume Labels
+
+It is possible to set labels for all newly created volumes. By default, all volumes are labeled as follows:
+
+* `kubernetes.io/created-for/pvc/name`
+* `kubernetes.io/created-for/pvc/namespace`
+* `kubernetes.io/created-for/pv/name`
```

Comment:
These are not [well-known labels](https://kubernetes.io/docs/reference/labels-annotations-taints/) under `kubernetes.io`. I would prefer to use something else. Either the labels from the csi `external-provisioner` or custom ones under `csi.hetzner.cloud`.

- `csi.storage.k8s.io/pvc/name`
- `csi.storage.k8s.io/pvc/namespace`
- `csi.storage.k8s.io/pv/name`

Or

- `csi.hetzner.cloud/pvc-name`
- `csi.hetzner.cloud/pvc-namespace`
- `csi.hetzner.cloud/pv-name`